cli_tools/gce_ovf_import/ovf_importer/ovf_importer.go

```diff
@@ -18,81 +18,75 @@ import (
 	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
 	daisycompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
-	"github.com/vmware/govmomi/ovf"
-	computeBeta "google.golang.org/api/compute/v0.beta"
-	"google.golang.org/api/compute/v1"
-	"google.golang.org/api/iterator"
-	"google.golang.org/api/option"
 )
 
 const (
-	ovfWorkflowDir             = "daisy_workflows/ovf_import/"
+	ovfWorkflowDir             = "ovf_import/"
+	createInstanceWorkflow     = ovfWorkflowDir + "create_instance.wf.json"
 	instanceImportWorkflow     = ovfWorkflowDir + "import_ovf_to_instance.wf.json"
 	machineImageImportWorkflow = ovfWorkflowDir + "import_ovf_to_machine_image.wf.json"
 	logPrefix                  = "[import-ovf]"
-)
-
-const (
-	//Alpha represents alpha release track
-	Alpha = "alpha"
-
-	//Beta represents beta release track
-	Beta = "beta"
 
-	//GA represents GA release track
-	GA = "ga"
+	// Amount of time required after disk files have been imported. Used to calculate the
+	// timeout budget for disk file import.
+	instanceConstructionTime = 10 * time.Minute
 )
 
 // OVFImporter is responsible for importing OVF into GCE
 type OVFImporter struct {
-	ctx                   context.Context
-	storageClient         domain.StorageClientInterface
-	computeClient         daisycompute.Client
-	tarGcsExtractor       domain.TarGcsExtractorInterface
-	mgce                  domain.MetadataGCEInterface
-	ovfDescriptorLoader   ovfdomain.OvfDescriptorLoaderInterface
-	bucketIteratorCreator domain.BucketIteratorCreatorInterface
-	Logger                logging.Logger
-	zoneValidator         domain.ZoneValidatorInterface
-	gcsPathToClean        string
-	workflowPath          string
-	diskInfos             *[]ovfutils.DiskInfo
-	params                *ovfimportparams.OVFImportParams
-	imageLocation         string
-
-	// BuildID is ID of Cloud Build in which this OVF import runs in
-	BuildID string
+	ctx                 context.Context
+	storageClient       domain.StorageClientInterface
+	computeClient       daisycompute.Client
+	multiImageImporter  ovfdomain.MultiImageImporterInterface
+	tarGcsExtractor     domain.TarGcsExtractorInterface
+	ovfDescriptorLoader ovfdomain.OvfDescriptorLoaderInterface
+	Logger              logging.Logger
+	gcsPathToClean      string
+	workflowPath        string
+	diskInfos           *[]ovfutils.DiskInfo
+	params              *ovfdomain.OVFImportParams
+	imageLocation       string
+	paramValidator      *ParamValidatorAndPopulator
+
+	// Populated when disk file import finishes.
+	imageURIs []string
 }
 
 // NewOVFImporter creates an OVF importer, including automatically populating dependencies,
-// such as compute/storage clients.
-func NewOVFImporter(params *ovfimportparams.OVFImportParams) (*OVFImporter, error) {
+// such as compute/storage clients. workflowDir is the filesystem path to `daisy_workflows`.
+func NewOVFImporter(params *ovfdomain.OVFImportParams, workflowDir string) (*OVFImporter, error) {
```

Comment:
There's another comment on whether workflowDir should be a flag or not, but not considering that question, can workflowDir be a part of OVFImportParams (it already contains non flags)?

Reply:
Good point -- moved to `OVFImportParams`